### ISSUE TITLE
Overwolf (Pseudo Dota+) must be Banned from Dota 2.

### ISSUE THREAD

**Issue by DeckCardCain** [ValveSoftware/Dota2-Gameplay#15564]
### Description

It gives unfair advantages for people that download a Third Party App that isn't allowed to be used in tournaments.
It takes subscribers away from Dota+
It makes all Dota+ basically suckers for supporting Valve and Dota instead of using a Third Party App.
It has copies infested with malware (I'm not saying the official client, from the official website, is a malware) around the web...
It leads for accounts falling victim to scams and losing their stuff.

It is simple, really: Make an announcement that Overwolf won't be tolerated anymore. Give a time for people to spread the information so no accidental bans happens, and then start with the prohibition.

If Overwolf changes to runs in the background in disguise, it's clearly cheating and bannable.

**I am making a formal accusation here**: I believe that, some of the owners of Overwolf, are actually Valve Employees: that is the only reasonable explanation why Valve would allow a third party software that bypass privacy settings to go for this long.

The blame is entirely on the Dota 2 Developers team, specially the head of it, Icefrog, for allowing such cheating software to undermine the quality of the matchmaking and undermine sales of Dota+. It is self sabotage. So it makes sense to say someone at Valve is actually making a profit out of Overwolf.

And don't come with "If you ban Overwolf, other will take it's place". It's Valve job to decide what is, and what isn't cheating, and they have the freedom to ban any third app that gives unfair advantage.

**Comment by Memoframe** [comment]
When ingame Dota+ gave advantages for years you did not care.
Jokes aside, who even cares about Overwolf, which, afaik, do nothing to game client, if things like this exists:
https://www.youtube.com/watch?v=RwzIq04vd0M

**Comment by TobyFlendersonFromHR** [comment]
> I am making a formal accusation here: I believe that, some of the owners of Overwolf, are actually Valve Employees

what a WILD ACCUSATION!

i hope you follow this up with some investigative journalism

**Comment by NekoPara1477** [comment]
I agree with you, I from steam, and I think all users overplus want be punished and get VAC on steam accounts

**Comment by NekoPara1477** [comment]
They also supported the melonity cheat

**Comment by NekoPara1477** [comment]
![Uploading Screenshot_20240221_165639_com.yandex.browser.jpg…]()

**Comment by NekoPara1477** [comment]
![Screenshot_20240221_165605_com yandex browser](https://github.com/ValveSoftware/Dota2-Gameplay/assets/160736798/1f139408-a0f0-4ddb-bec2-e77bc3861450)

**Comment by NekoPara1477** [comment]
![Screenshot_20240221_165639_com yandex browser](https://github.com/ValveSoftware/Dota2-Gameplay/assets/160736798/3373320e-8ed9-4dfc-b887-e63736982719)

**Comment by DeckCardCain** [comment]
> When ingame Dota+ gave advantages for years you did not care. Jokes aside, who even cares about Overwolf, which, afaik, do nothing to game client, if things like this exists: https://www.youtube.com/watch?v=RwzIq04vd0M

I have, ALWAYS, requested that the Avoid List from Dota+ (and Dota+, TBH) to be Free, since it's a great quality of life enhancement for the game. It adds sub-quests, a lot of data and enhancements to the interface.

It's selective matchmaking, therefore is a form of pay to win. Just because I am posting this here now, doesn't mean I have not posted about this subject before on the official Discussion Board.

> > I am making a formal accusation here: I believe that, some of the owners of Overwolf, are actually Valve Employees
> 
> what a WILD ACCUSATION!
> 
> i hope you follow this up with some investigative journalism

I hope you get better at making jokes instead of trolling around GitHub. Your nickname suits you perfectly.

> ![Screenshot_20240221_165605_com yandex browser](https://private-user-images.githubusercontent.com/160736798/306612964-1f139408-a0f0-4ddb-bec2-e77bc3861450.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg1MTk1NzAsIm5iZiI6MTcwODUxOTI3MCwicGF0aCI6Ii8xNjA3MzY3OTgvMzA2NjEyOTY0LTFmMTM5NDA4LWEwZjAtNGRkYi1iZWMyLWU3N2JjMzg2MTQ1MC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyMVQxMjQxMTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNTNjNGNhODdlOWYzYzcwM2JlODg2NjFhMDk4YjhlOWRmZGIwNmU2MDgzNmM0OWNlNDE3ZjcxMzA5MGE5YWFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Td7ZVBF5eIbMv-SinfZIqW9iRdZBFa7YqE5GIz91tO0)

This.

**Comment by Memoframe** [comment]
> > When ingame Dota+ gave advantages for years you did not care. Jokes aside, who even cares about Overwolf, which, afaik, do nothing to game client, if things like this exists: https://www.youtube.com/watch?v=RwzIq04vd0M
> 
> I have, ALWAYS, requested that the Avoid List from Dota+ (and Dota+, TBH) to be Free, since it's a great quality of life enhancement for the game. It adds sub-quests, a lot of data and enhancements to the interface.
> 
> It's selective matchmaking, therefore is a form of pay to win. Just because I am posting this here now, doesn't mean I have not posted about this subject before on the official Discussion Board.

You always said but used it I assume?
If not I'm sorry.
I never used Dota+, neither Overwolf nor official one, even when it was free and I do not care about those who use it.
Oficially providing advantages to players and gating it behind money will never be an option for community.
Also most likely it's not allowed to gather info about user (processes running on machine for example), just a guess ofc but most likely it's true, atleast for EU.
While software is not injecting into game it's ok unfortunately.
Imo scripts/macros are worse, even just mouse scripts, not even talking about ingame ones or software which can do it even better and without injecting.
But maybe I'm wrong? 
Overwolf has something even worse?
You did not describe a reason to ban it tbh.
You say advantage but do not say what advantage.
I'm curious now.

Anyway it's all does not matter, because it's bugtracker.
Not a forum.
Not a support.
It's a place where people say "this is working wrong when I do this", so devs can fix it.
If you wonna discuss it with me further we can move to PM, I do not wonna argue in bugtracker, at least not so much :D

**Comment by DeckCardCain** [comment]
> > > When ingame Dota+ gave advantages for years you did not care. Jokes aside, who even cares about Overwolf, which, afaik, do nothing to game client, if things like this exists: https://www.youtube.com/watch?v=RwzIq04vd0M
> > 
> > 
> > I have, ALWAYS, requested that the Avoid List from Dota+ (and Dota+, TBH) to be Free, since it's a great quality of life enhancement for the game. It adds sub-quests, a lot of data and enhancements to the interface.
> > It's selective matchmaking, therefore is a form of pay to win. Just because I am posting this here now, doesn't mean I have not posted about this subject before on the official Discussion Board.
> 
> You always said but used it I assume? If not I'm sorry. I never used Dota+, neither Overwolf nor official one, even when it was free and I do not care about those who use it. Oficially providing advantages to players and gating it behind money will never be an option for community. Also most likely it's not allowed to gather info about user (processes running on machine for example), just a guess ofc but most likely it's true, atleast for EU. While software is not injecting into game it's ok unfortunately. Imo scripts/macros are worse, even just mouse scripts, not even talking about ingame ones or software which can do it even better and without injecting. But maybe I'm wrong? Overwolf has something even worse? You did not describe a reason to ban it tbh. You say advantage but do not say what advantage. I'm curious now.
> 
> Anyway it's all does not matter, because it's bugtracker. Not a forum. Not a support. It's a place where people say "this is working wrong when I do this", so devs can fix it. If you wonna discuss it with me further we can move to PM, I do not wonna argue in bugtracker, at least not so much :D

Yes, I have been using Dota + since it was released: I am not an hypocrite, and you don't have to be apologize for that. But the main reason was the relics and the exclusives sets - that could not be bought before. My avoid list is small, because I use most of my shards for the cosmetics Steam Shard offers.

The advantage it provides is this: it allows you to see which heroes the players have been picking and having most success with, even if the player has Privacy settings on. That means you can pre-emptively ban, pick or counter those heroes - it bypass the Privacy Settings but not only that, it uses a unique HUD Interface to display those things. For that reason, alone, it gives an unfair advantage for using this third party app.

I understand your concerns, and I respect your opinion about the usage of the bug tracker. But, if it's bypassing the "Privacy Settings", it is a bug with the "Privacy Settings".

The thing is, it collects data from the game API, same way Dotabuff or OpenCloud. When a third app party allows you to instantly gather intel during picks, it is not the same - at all - as opening Dotabuff and checking the profile of the peoeple you are playing against: the app does it for you, instantly, for the 5 enemies and 4 allies.

I'm not arguing at all my friend, I respect your point and the way you write: you have been very polite.

I posted this under "Other", unlike the other post I made about "Lifestealer Aghs interaction with Bristleback and Mars passive skills. That post, clearly a tooltip clarification must be made, since I tested on the demo myself and elaborated on the post. You can check it out if you want and tips are welcome, in case I missed anything.

Remember back when people would mod the game, so the trees would disappear, and show just a circle instead? For a long time, that was allowed, and it gave an unfair advatage to people that used it. Even streamers did use it. Unlike Dota+, I never used this, and a lot of people complained, not just me, so Valve eventually forbid it.

Overwolf also allows you add "hud interface improvements", as an addon in World of Warcraft, to always display an extra window, that basically shows on your screen a bar similar to the one you see when you have your hero, or another hero selected. This allows you to see the HP, Mana, Inventory and Cooldowns of the hero you choose to set up in your screen, allowing you to know all that information without clicking  on the specific hero.

There are many things that you can say provides advantage for the have and the have nots:

Ultra-Wide resolution expand your screen, so you get more open space between the different aspects of the hud. But it's a piece of hardware, and Valve decided to add support for this. So it falls under the same category as "having a mouse that works instead of a mouse that double click accidentaly" or "having a computer that runs on high fps". Sadly, Dota+ falls under the same category: and since the game is international, it can get very expensive since the price is not regionalized. For me, it costs the equivalent of 3 Litres of Gas a month.

I went a little off topic there, I am sorry. 
But I think I explained why Overwolf gives an unfair advantage bypassing Privacy Settings In-Game and, since it abuses an unfixable bug (more of a feature, Dota API) and by providing extra user interface tools - you can set icons on your screen to display the cooldown of all enemies BKB, for instance.

Thanks for your participation, I hope I have clarified why I posted this on Bug Tracker.

**Comment by TobyFlendersonFromHR** [comment]
yeah sweet novel bro

**Comment by DeckCardCain** [comment]
> yeah sweet novel bro

Thanks for spamming, trolling and harassing. Reported and Blocked.

**Comment by kaisenbaka** [comment]
agree, i experienced a previous griefer that has beef with me, somehow when we were in different team, he precisely picked 3 same hero as me (I usually pick at that rotation if the highest priority hero was banned example: Es->Zues->tusk). i was so mad

**Comment by MoundN** [comment]
"I am making a formal accusation here: I believe that, some of the owners of Overwolf, are actually Valve Employees:"

You my friend have severe brain damage. 

Please refrain from further commenting on topics you're absolutely clueless about, i'm not expecting everyone to exercise common sense but if people refuse to do so they need to just stfu.

**Comment by rhyanwong7-cmd** [comment]
your so good at saving dota what about people who works hard collecting items who got their item hijack been 12 years of collecting items all gone wipe out i support overplus for giving the same vipe as what i have before fuck you  
DeckCardCain for protecting vale so much when they dont punish the scammer or hijackers fuck you real hard. get out of the overplus you dont belong here

**Comment by MoundN** [comment]
> your so good at saving dota what about people who works hard collecting items who got their item hijack been 12 years of collecting items all gone wipe out i support overplus for giving the same vipe as what i have before fuck you DeckCardCain for protecting vale so much when they dont punish the scammer or hijackers fuck you real hard. get out of the overplus you dont belong here

As eluded to before, you have severe braindamaged and should hence refrain from commenting about this topic. I think baby shark is more up to your speed.
